services/honeycomb-agent/config.yaml

```diff
@@ -25,7 +16,7 @@ watchers:
     processors:
       - sample:
           type: static
-          rate: 10 # 1/10 sample rate
+          rate: 0.2 # 1/10 sample rate
```

Comment:
I'm not sure the syntax here. did `rate: 10` mean `10/100`? And so .2 means `.2/100`?
Should the comment here be updated accordingly?

Reply:
Thanks, fixed.